src/vars.h

```diff
@@ -278,8 +301,15 @@ static inline void SwitchToOldLVarsAndFree(Obj old, const char *file, int line)
 **
 **  'ASS_LVAR' assigns the value <val> to the local variable <lvar>.
 */
-#define ASS_LVAR(lvar,val)      (STATE(PtrLVars)[(lvar)+2] = (val))
-#define ASS_LVAR_WITH_CONTEXT(context,lvar,val)      (ADDR_OBJ(context)[(lvar)+2] = (val))
+static inline void ASS_LVAR(UInt lvar, Obj val)
+{
+    STATE(PtrLVars)[lvar + 2] = val;
+}
+
+static inline void ASS_LVAR_WITH_CONTEXT(Obj context, UInt lvar, Obj val)
+{
+    ADDR_OBJ(context)[lvar + 2] = val;
```

Comment:
Could this in principle also be a `CONST_ADDR_OBJ` like in `OBJ_LVAR_WITH_CONTEXT` below?